### ISSUE TITLE
decrypt string in flutter dart 

### ISSUE THREAD

**Issue by preetideswal** [leocavalcante/encrypt#229]
Java Code: 
        public String decrypt(String encryptedText) throws Exception {
        try {
            byte[] decryptedTextBytes = null;
            byte[] encryptedTextBytes = Base64.decode(encryptedText, Base64.NO_WRAP);
            final SecretKeySpec secretSpec = sha256("myCheckSumValue");
            decryptedTextBytes = decrypt(secretSpec, ivBytes, encryptedTextBytes);
            return new String(decryptedTextBytes, StandardCharsets.UTF_8);
        } catch (Exception e) {
            msg_display.setText("QR code was not scanned properly  ! ! ! ! ! !");
            throw new RuntimeException(e);
        }
    }
    public SecretKeySpec sha256(String base) {
        try {
            final MessageDigest digest = MessageDigest.getInstance("SHA-256");
            byte[] hash = base.getBytes(StandardCharsets.UTF_8);
            digest.update(hash, 0, hash.length);
            byte[] key = digest.digest();
            return new SecretKeySpec(key, "AES");
        } catch (Exception ex) {
            throw new RuntimeException(ex);
        }
    }
    public byte[] decrypt(final SecretKeySpec key, final byte[] iv, final byte[] decodedCipherText) throws Exception {
        final Cipher cipher = Cipher.getInstance("AES/CBC/PKCS5Padding");
        IvParameterSpec ivSpec = new IvParameterSpec(iv);
        cipher.init(Cipher.DECRYPT_MODE, key, ivSpec);
        return cipher.doFinal(decodedCipherText);
    }

But i don't know how i decrypt in flutter 

Thankyou

**Comment by yogithesymbian** [comment]
yes, there is no documentation about to decrypted of string encrypter aes.
